README.md

```diff
@@ -1,87 +1,81 @@
-
-This repo contains the [JupyterLite](https://jupyterlite.readthedocs.io) config
-files used by the new SymPy Live. The code is deployed as a static site to GitHub Pages.
+# JupyterLite Shell for SymPy Live 
 
+This repo contains the code that powers the [SymPy Live Shell](https://live.sympy.org/),
+which is based on the [JupyterLite](https://jupyterlite.readthedocs.io) application,
+and deployed as a static site to GitHub Pages.
 
 
 ## ✨ Try it in your browser ✨
 
 - **https://live.sympy.org/**
 
-Alternative URLs:
+Other interesting URLs:
 - https://www.sympy.org/en/shell.html
 - [evaluate command](https://www.sympy.org/en/shell.html?evaluate=diff(sin(x)%2C%20x)%0A%23--%0A)
 
 
-## Requirements
-
-JupyterLite is being tested against modern web browsers:
-
-- Firefox 90+
-- Chromium 89+
-
 
+## Developer information
 
-## Commands
+The SymPy Live shell is a thin wrapper around the `repl` app in the JupyterLite.
+In order to reduce the maintentance needs,
+we leverage the code from the [`jupyterlite/demo`](https://github.com/jupyterlite/demo) project,
+and provide only minimal customization: a custom landing page that loads the REPL in an iframe.
 
-See Developer instructions below for installing the prerequisites.
 
-Run jupyter lite locally:
-```bash
-jupyter lite serve
-```
+### Configs
 
-Build ready for distribution via static serve:
-
-```bash
-jupyter lite build
-```
-
-The files will be saved in `_output/`.
+The operation of JupyterLite is configured in several places:
+- `requirements.txt`: specifies the version `jupyterlab`, `jupyterlite-core` and `jupyterlite-pyodide-kernel`.
+- `jupyter_lite_config.json`: general configs for the jupyter lite build system
+- `repl/jupyter-lite.json`: configs specific to the REPL app. See [example here](https://github.com/ivanistheone/live/blob/357e60a228b43ac28ef835953d00f4495a429d78/repl/jupyter-lite.json).
+- `overrides.json`: customizations of the JupyterLite UI
 
+In addition to the above, the SymPy Live shell depends on the following customizations:
+- Custom landing page [index.html](https://github.com/sympy/live/blob/main/templates/index.html) that embeds the JupyterLite REPL.
+- Automatic import `from sympy import *` and init commands in [index.html template](https://github.com/sympy/live/blob/main/templates/index.html#L6-L11).
+- JavaScript code to decode evaluate query string, see [here](https://github.com/sympy/live/blob/main/templates/index.html#L49-L56).
 
-You can test the files using any static server, for example:
+> [!NOTE]
+> Updating the version of SymPy running on [live.sympy.org/](https://live.sympy.org/) usually requires updating the version of `jupyterlab`, `jupyterlite-core`, and `jupyterlite-pyodide-kernel` listed in `requirements.txt`. Use the [upstream requirements file](https://github.com/jupyterlite/demo/blob/main/requirements.txt) as reference. See [here](https://pyodide.org/en/stable/usage/packages-in-pyodide.html) for the backage verison included in Pyodide. 
```

Comment:
I think it would be good to mention here that the first step is updating the version in pyodide itself. Perhaps a link to this PR would show how this looks: https://github.com/pyodide/pyodide/pull/5098.

Then only after pyodide actually makes a new release it becomes possible to update the jupyterlite version so that the website receives the new version of sympy. (I was confused when that PR got merged but the website did not immediately update. It is now several months later so maybe worth setting expectations clearly here.)